### DISCUSSION TITLE
Settings reset everytime I launch the game

### DISCUSSION THREAD

**Discussion by Saraistupid** [ppy/osu#30944] [Q&A] [answered]
For a while now, everytime I launch Lazer the video and audio settings get reset, and ever since yesterday, the game launches at the first time setup wizard
The game files are on an NTFS partition, had to move them out of my EXT4 partition due to space issues.
Running Arch Linux and Lazer 2024.1115.3

**Comment by Saraistupid** [comment]
I think I found the cause, the game doesn't use the same config file on launch, but I don't know what the root issue is
![image](https://github.com/user-attachments/assets/86f7544a-c2ab-439b-8c0f-2caba8dce3a6)

**Reply by peppy** [reply]
Those files are temporary files. They should be cleaned up when the final write happens.

**Comment by Saraistupid** [comment, marked as answer]
Fixed the problem, I had to delete all the extra files the game has created, and the `framework.ini` and `game.ini` files, then repaired the NTFS partition using Windows.
